avocado/core/test.py

```diff
@@ -133,7 +133,156 @@ def str_filesystem(self):
                                  % (self.str_uid, str(self)))
 
 
-class Test(unittest.TestCase):
+class BaseTestData(object):
```

Comment:
This is rather cosmetic, but it makes only a little sense to have `BaseTestData` and `TestData` classes, when the main `avocado.Test` inherits from `TestData` and __ALL__ tests must be inherited from it. Therefor the only available API is the `TestData`.